client/me/account/main.jsx

```diff
@@ -302,42 +300,6 @@ class Account extends Component {
 		return true;
 	}
 
-	communityTranslator() {
-		if ( ! this.shouldDisplayCommunityTranslator() ) {
-			return;
-		}
-		const { translate } = this.props;
-		return (
-			<FormFieldset>
-				<FormLegend>{ translate( 'Community Translator' ) }</FormLegend>
-				<FormLabel htmlFor={ ENABLE_TRANSLATOR_KEY }>
-					<FormCheckbox
-						checked={ this.getUserSetting( ENABLE_TRANSLATOR_KEY ) }
-						onChange={ this.updateCommunityTranslatorSetting }
-						disabled={ this.getDisabledState( INTERFACE_FORM_NAME ) }
-						id={ ENABLE_TRANSLATOR_KEY }
-						name={ ENABLE_TRANSLATOR_KEY }
-						onClick={ this.getCheckboxHandler( 'Community Translator' ) }
```

Comment:
If I understand it correctly, the `getCheckboxHandler` isn't used anywhere. Considering that you extracted it to a component, maybe it would be worth removing it as well.

Reply:
You are right! Just removed it.